_posts/2019-04-25-safer-and-cleaner-seas-stm.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: post
+title: Safer and cleaner seas via Sea Traffic Management
+author: arto
+excerpt: Air traffic has been digitalized long ago. Now, the seas are going same route as well. Future holds much safer and cleaner marine traffic for all of us.
+tags:
+- Environment
+- Automation
+- Digitalization
+- Project
+- Distributed
+- Java
+- XML
+---
+**Human errors can be expensive**. When you are making decisions, and acting on them while not fully concentrating, it leaves room for mistakes. A typical human error might be writing your address wrong when ordering an item. Or copying a single digit of bank account number wrong, from a handwritten note. Perhaps you are acting on too little information, or are in a hurry. Those mistakes happen every day to someone. In most cases you can recover from them through some extra work, no harm done.
+
+![Accidents at sea (source listed at end of the article)](/img/safer-and-cleaner-seas/marine_casualty_information.png)
+
+Mistakes that happen when operating a moving vehicle of any kind, such as a car, train, ship or an airplane, tend to be much more severe. These mistakes are not just about mild embarrassment or some monetary cost. **Any mistakes involving a moving vehicle bring much more costs to recover**. Sometimes they may result in environmental disasters, bodily harm, or even loss of life. Doing routine work that has to be repeated just right is a major reason for human errors.
+
+This is why any kind of professional transportation has multiple overlapping safety controls in place to prevent any accidents. Still, occasionally things manage to go wrong, and accidents do happen. Sometimes it's a matter of failing equipment or parts, but often it can also be attributed to mistakes in judgment, not being informed enough, or trusting bad information.
```

Comment:
things **can** go wrong

Reply:
Jatkan mpajunen ideaa

Still, occasionally things can go wrong and accidents do happen.

Reply:
"Sometimes it's a matter of failing equipment or parts, but often it can also..."
Varmempi jättää "it's" käyttämättä ja kirjoittaa auki. Ei tarvitse pilkkua.

"Sometimes it is a matter of failing equipment or defective parts but often it can also..."